src/main/java/net/pms/dlna/RealFile.java

```diff
@@ -180,7 +180,6 @@ public synchronized void resolve() {
 
 						if (medias.size() == 1) {
 							setMedia(medias.get(0));
-							getMedia().postParse(getType(), input);
```

Comment:
@valib I've made this branch up to date with master now, but I'm not sure about this line since `postParse` does more than just set the MIME type. It can also do `secondaryFormatValid = false` and check for external subtitles. Are you sure we want to stop that happening here?
I don't know but I just wanted to ask to make sure it has been considered.